ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-SDO-nonprod.json

```diff
@@ -79,5 +79,20 @@
         "CRUD": "R"
       }
     ]
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "NotSuitable_SDO",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "caseworker-civil-admin",
```

Comment:
might as well just make this "judge-profile" and "legal-adviser" only otherwise you'll just have to come back and change this later anyway, but up to you how you want to do it